index.html

```diff
@@ -16,131 +17,159 @@
-                    </ul>
-                    <button class="detail-btn">See project</button>
+                        <p class="detail-text">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required.</p>
+                        <ul class="btns">
+                            <li class="btn">HTML</li>
+                            <li class="btn">CSS</li>
+                            <li class="btn">JavaScript</li>
+                        </ul>
+                        <button class="detail-btn">See project</button>
+                    </div>
                 </div>
                 <div class="projects">
                     <img src="assets/images/Snapshoot Portfolio-4.png" alt="project 1" class="project-image">
-                    <div class="project-details">
-                        <h2 class="project-tile">Multi-Post
-                            Stories</h2>
-                        <div class="stack-year">
-                            <h3 class="title">CANOPY</h3>
-                           <img src="assets/icons/Counter.png" alt="counter">
-                           <h4 class="lang-year">Back End Dev</h4>
-                           <img src="assets/icons/Counter.png" alt="counter">
-                           <h4 class="lang-year">2015</h4>
-                        </div>
+                    <div>
+                        <div class="project-details">
+                            <h2 class="project-tile">Multi-Post
+                                Stories</h2>
+                            <div class="stack-year">
+                                <h3 class="title">CANOPY</h3>
+                                <img src="assets/icons/Counter.png" alt="counter">
+                                <h4 class="lang-year">Back End Dev</h4>
+                                <img src="assets/icons/Counter.png" alt="counter">
+                                <h4 class="lang-year">2015</h4>
+                            </div>
                         </div>
-                    <p class="detail-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul class="btns">
-                        <li class="btn">HTML</li>
-                        <li class="btn">CSS</li>
-                        <li class="btn">JavaScript</li>
-                    </ul>
-                    <button class="detail-btn">See project</button>
+                        <p class="detail-text">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required.</p>
+                        <ul class="btns">
+                            <li class="btn">HTML</li>
+                            <li class="btn">CSS</li>
+                            <li class="btn">JavaScript</li>
+                        </ul>
+                        <button class="detail-btn">See project</button>
```

Comment:
- Kindly make sure your project's button has the same proportions of size with the Figma design :recycle: 
Your design:
![Screenshot from 2023-03-30 11-19-35](https://user-images.githubusercontent.com/97834160/228790394-64fe1f9b-562a-4942-b873-a57fe9931b6b.png)

Figma:
![Screenshot from 2023-03-30 11-15-22](https://user-images.githubusercontent.com/97834160/228790302-72a75535-8350-4acc-9ce4-0da8b9d5848d.png)